__tests__/script.integration.test.js

```diff
@@ -1,31 +1,47 @@
 const fs = require("fs");
 const puppeteer = require("puppeteer");
-const { beforeAll, describe, expect, test } = require("@jest/globals");
+const {
+  afterAll,
+  beforeAll,
+  describe,
+  expect,
+  test,
+} = require("@jest/globals");
+
+let browser;
```

Comment:
We now set the browser once for all tests. This makes the tests take 5 seconds compared to 12.